packages/react-native-select-pro/src/components/select/index.tsx

```diff
@@ -94,43 +102,56 @@ export const Select = forwardRef((props: SelectProps, ref: ForwardedRef<SelectRe
     const { aboveSelectControl } = openedPosition;
     const selectedOptionTyped = selectedOption as OptionType;
 
+    const firstRenderRef = useRef(false);
```

Comment:
create global hook for first render, sth like this: https://usehooks-ts.com/react-hook/use-is-first-render